examples/one-page.html

```diff
@@ -115,27 +114,43 @@
       }`;
 
       const Presentation = () => html`
-        <${Deck} theme=${theme} template=${template}>
+        <${Deck} theme=${theme} template=${template} transitionEffect="fade">
           <${Slide}>
             <${FlexBox} height="100%">
               <${SpectacleLogo} size=${500} />
             </${FlexBox}>
           </${Slide}>
           <${Slide}>
             <${FlexBox} height="100%" flexDirection="column">
-              <${Heading} fontSize="150px">SPECTACLE</${Heading}>
-              <${Heading} fontSize="h2">A ReactJS Presentation Library</${Heading}>
-              <${Heading} color="primary" fontSize="h3">Where you can write your decks in JSX, Markdown, or MDX!</${Heading}>
+              <${Heading} margin="0px" fontSize="150px">SPECTACLE</${Heading}>
+              <${Heading} margin="0px" fontSize="h2">A ReactJS Presentation Library</${Heading}>
+              <${Heading} margin="0px 32px" color="primary" fontSize="h3">Where you can write your decks in JSX, Markdown, or MDX!</${Heading}>
             </${FlexBox}>
             <${Notes}>
               <p>Notes are shown in presenter mode. Open up localhost:3000/?presenterMode=true to see them.</p>
             </${Notes}>
           </${Slide}>
-          <${Slide}>
+          <${Slide} transitionEffect="slide">
             <${Heading}>Code Blocks</${Heading}>
-            <${CodePane} fontSize=${18} language="cpp" autoFillHeight>
-              ${cppCodeBlock}
-            </${CodePane}>
+            <${Stepper} defaultValue=${[]} values=${[
+        [1, 1],
+        [23, 25],
+        [40, 42]
+      ]}>
+              ${(value, step) => html`<${Box} position="relative">
+                <${CodePane} highlightStart=${value[0]} highlightEnd=${
+                value[1]
+              } fontSize=${18} language="cpp" autoFillHeight>
+                  ${cppCodeBlock}
```

Comment:
Indentation is off for code block in display. Not sure exactly where (auto-trim), but I'll take a look.

Reply:
It's getting `margin-left: 0px; flex: 1 1 0%; padding-left: 0.25em;` for the first line then `margin-left: 65.0215px; flex: 1 1 0%; padding-left: 0.25em;` for all subsequent lines in rendered HTML

Reply:
Actually, looks like this is a problem from upstream as it's also in latest published version: https://unpkg.com/spectacle@6.0.0-alpha.8/examples/one-page.html?slide=2&slideElement=-1

Reply:
Captured in https://github.com/FormidableLabs/spectacle/issues/846